src/pages/components/combobox.mdx

```diff
@@ -0,0 +1,233 @@
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To filter down a list of options</li>
+      <li>To quickly find a known option</li>
+      <li>As a search field</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          For a searchable dropdown, use [Autocomplete](/components/autocomplete) instead
+        </Markdown>
+      </li>
+      <li>
+        <Markdown>
+          To make more than one selection, use [Multiselect](/components/multiselect) instead
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+As the user types, options appear in the list box that match what the user is typing.
+
+import ComboboxDefault from '../../examples/dropdowns/combobox/Default.tsx';
+import ComboboxDefaultCode from '!!raw-loader!../../examples/dropdowns/combobox/Default.tsx';
+
+<CodeExample code={ComboboxDefaultCode}>
+  <ComboboxDefault />
+</CodeExample>
+
+## Customization
+
+### Size
+
+Combobox can be displayed in either default or compact size.
+
+import ComboboxSize from '../../examples/dropdowns/combobox/Size.tsx';
+import ComboboxSizeCode from '!!raw-loader!../../examples/dropdowns/combobox/Size.tsx';
+
+<CodeExample code={ComboboxSizeCode}>
+  <ComboboxSize />
+</CodeExample>
+
+### Hidden Label
+
+The label can be hidden.
+
+import ComboboxHidden from '../../examples/dropdowns/combobox/Hidden.tsx';
+import ComboboxHiddenCode from '!!raw-loader!../../examples/dropdowns/combobox/Hidden.tsx';
+
+<CodeExample code={ComboboxHiddenCode}>
+  <ComboboxHidden />
+</CodeExample>
+
+### Selection Logic
+
+The combobox can be customized to include selection logic that highlights the first relevant
+result as the user types, allowing the user to press Enter to select the item.
+If this logic is not included, the user must manually select a result from the list.
+
+import ComboboxSelection from '../../examples/dropdowns/combobox/Selection.tsx';
+import ComboboxSelectionCode from '!!raw-loader!../../examples/dropdowns/combobox/Selection.tsx';
+
+<CodeExample code={ComboboxSelectionCode}>
+  <ComboboxSelection />
+</CodeExample>
+
+## Usage
+
+<BestPractice rows>
+  <Do
+    imageIsFreeForm
+    imageSource={props.data.comboboxIconPlacementDo.childFile.childImageSharp.gatsbyImageData}
+  >
+    <p>
+      Always place the magnifying glass icon at the start of the input when using the Combobox for
+      search.
+    </p>
+  </Do>
+  <Do
+    imageIsFreeForm
+    imageSource={props.data.comboboxHighlightDo.childFile.childImageSharp.gatsbyImageData}
+  >
+    <p>
+      When using Combobox as a search input, automatically highlight the first suggestion in the
+      list box and allow the user to press the Enter key to submit the search.
+    </p>
+  </Do>
+  <Dont
+    imageIsFreeForm
+    imageSource={props.data.comboboxSearchableDont.childFile.childImageSharp.gatsbyImageData}
+  >
+    <p>
+      <Markdown>
+        Use an [Autocomplete](/components/autocomplete) component when creating a searchable
```

Comment:
@Francois-Esquire  Replacing the problematic "searchable dropdown" phrase and changing the content to this:

"Use an Autocomplete component when creating a dropdown that shows a list of options without needing to enter text in the input field."

Reply:
For sure, just updated.